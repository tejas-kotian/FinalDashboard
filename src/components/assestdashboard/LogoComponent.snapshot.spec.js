import React from "react";
import LogoComponent from "./LogoComponent";
import renderer from "react-test-renderer";



it("sets submit button label 'Saving...' when saving is true", () => {
  const tree = renderer.create(
    <LogoComponent></LogoComponent>
  );

  expect(tree).toMatchSnapshot();
});

