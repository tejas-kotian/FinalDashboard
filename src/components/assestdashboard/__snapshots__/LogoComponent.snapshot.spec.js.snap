// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sets submit button label 'Saving...' when saving is true 1`] = `
<div
  className="small mt-5"
>
  <div
    className="d-flex justify-content-center"
  >
    <test-file-stub />
  </div>
  <div
    className="d-flex justify-content-center mr-5 medium"
  >
    Please input desired allocation into available startegies to enable analysis and further actions  
  </div>
</div>
`;
